packages/workbox-cli/src/lib/run-wizard.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+const fse = require('fs-extra');
+const ol = require('common-tags').oneLine;
+
+const askQuestions = require('./questions/ask-questions');
+const logger = require('./logger');
+
+module.exports = async () => {
+  const config = await askQuestions();
+  const {configLocation} = config;
+  delete config.configLocation;
+
+  const contents = `module.exports = ${JSON.stringify(config, null, 2)};`;
+  await fse.writeFile(configLocation, contents);
+
+  logger.log(`All done! To build your service worker, run
```

Comment:
Nit: "to rebuild"

Reply:
"Rebuild" isn't accurate, since I am not triggering the build process following the completion of the wizard's questions.

Instead, I'm displaying this message because I want to push developers in the direction of modifying their build scripts to add in the `workbox generateSW` step, which ensures that they get a fresh precache payload each time they rebuild their web app.

My concern is that running the build following the wizard completion process makes it seem like it's a one-off thing that's now done, rather than something that needs to be done repeatedly.

Reply:
So if I run the wizard, I don't get a service worker? In which case we aren't 'All Done', there is next step that needs to be done. if I'm following.

Reply:
If you run the wizard, you get a configuration file that could then be used with `workbox generateSW path/to/config-file.js`.

"All done" refers to being all done with questions and having a usable configuration that's been written to disk. I'm happy to remove that if it comes across as confusing.

You still need to run workbox to generate a SW, ideally as part of your build process, which is what I am attempting to get across in that message.

In v3, `workbox --help` displays:

```sh
  Commands:
    wizard
      Runs the configuration wizard, which will generate a
      config file based on answers to questions.
```

which conveys that same message: run the wizard to create a config file.

Reply:
Yeah we need to change the wording of this.

Maybe something like "You are now ready to build your service worker, just run 'workbox generate-sw'.'".

Sorry I completely missed this.

Reply:
I've further tweaked the message after your initial feedback. It now looks like:

```sh
Your config has been saved. To build your service worker, run

  workbox generateSW ${configLocation}

as part of a build process. See https://goo.gl/8zZy1P for details.
You can further customize your service worker by making changes to ${configLocation}.
See https://goo.gl/YYPcyY for details.
```

With the first shortlink going to our npm scripts guide, and the second going to our configuration JSDocs. (Both can be updated if/when we get more relevant docs at different URLs, but those were the best places to point to as of right now.)

I really want to hit home the fact that making `workbox generateSW` part of your build is the recommended course of action. I also want to leave folks with guidance about how to modify the basic configuration, since confusion about what configuration options are supported seems to come up frequently.